### DISCUSSION TITLE
Docs say Frigate is running go2rtc 1.9.4. My logs say 1.9.2. What's going on?

### DISCUSSION THREAD

**Discussion by flyize** [blakeblackshear/frigate#15515] [Ask A Question]
As title suggests. At one point, I was using a custom version of go2rtc, but I don't see any go2rtc binary in appdata/frigate (running on Unraid). What am I missing?

**Comment by NickM-27** [comment]
Frigate briefly ran 1.9.4 before reverting, seems the docs just weren't changed back

**Comment by hawkeye217** [comment]
There was one place in the docs that still mentioned the incorrect version. I've updated it in https://github.com/blakeblackshear/frigate/pull/15513
